components/Tabs/src/private/MDCItemBarCell.m

```diff
@@ -562,6 +562,8 @@ - (void)updateInk {
   MDCInkView *inkView = _inkTouchController.defaultInkView;
   inkView.inkColor = _style.inkColor;
   inkView.inkStyle = _style.inkStyle;
+  inkView.usesLegacyInkRipple = NO;
+  inkView.clipsToBounds = inkView.inkStyle == MDCInkStyleBounded ? YES : NO;
```

Comment:
Nit: Might read better if you parenthesized the condition, but don't need to change it if it'll invalidate CI